runtime/vm/lookupmethod.c

```diff
@@ -939,7 +939,7 @@ defaultMethodConflictExceptionMessage(J9VMThread *currentThread, J9Class *target
 	Assert_VM_true(methodsLength > 1);
 
 	errorMsg = j9nls_lookup_message(J9NLS_DO_NOT_PRINT_MESSAGE_TAG | J9NLS_DO_NOT_APPEND_NEWLINE,
-			J9NLS_VM_DEFAULT_METHOD_CONFLICT_LIST, "conflicting default methods for '%2$s%4$s' in %6$s from classes [%7$s]");
```

Comment:
The default here is just wrong and should be `conflicting default methods for '%2$.*1$s%4$.*3$s' in %6$.*5$s from classes [%7$s]`

Reply:
The `2$`, `3$`, `#$` pattern is consistently removed from other NLS messages.  If you delete those `#$` patterns, you get the message I updated to